packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -114,9 +118,14 @@ export class Persona {
     return this.accountManager.accounts();
   }
 
-  // TODO implement
   public async getTxs(): Promise<ReadonlyArray<ProcessedTx>> {
-    return [];
+    if (!this.core) {
+      return [];
+    }
+
+    const processed = await Promise.all(this.core.signedAndPosted.value.map(s => this.processTransaction(s)));
+    const filtered = processed.filter(isNonNull);
```

Comment:
Why we can have null transactions?

Reply:
Because we cannot process every transaction. Only `SendTransacrtion` can be converted into `ProcessedTx` at the moment.